doc/getting_started/index.rst

```diff
@@ -227,6 +227,7 @@ directory using west:
          west init zephyrproject
          cd zephyrproject
          west update
+         west zephyr-export
```

Comment:
I think we need to update the section title. Currently it's "Get the source code" but that's not true any more since we also export Zephyr as a CMake package. It might be even better if we create a dedicated section "Export Zephyr as a CMake package".

Reply:
Very good point.
Will add it's own section.

Reply:
Done.

Reply:
of course. Thanks for noticing.